voice-output/voice-pack-voicer.js

```diff
@@ -44,7 +70,24 @@ class VoicePack {
       })(),
       loadAudioBuffer(audioContext, audioUrl),
     ]);
-    const voicePack = new VoicePack(syllableFiles, audioBuffer);
+    const [
+      actionFiles,
+    ] = await Promise.all([
+      (async () => {
+        const res = await fetch(indexUrl);
+        let j = await res.json();
+        j = j.filter(({name}) => /^actions\//.test(name));
+        return j;
+      })(),
+      loadAudioBuffer(audioContext, audioUrl),
+    ]);
+
+    const name=audioUrl.substring(
```

Comment:
IMO this should be a regex.

Reply:
removed, since we don't need `voiceActor` now